### DISCUSSION TITLE
Document how cookies are handled

### DISCUSSION THREAD

**Discussion by olalonde** [apify/got-scraping#96] [Q&A] [answered]
It would be good to have a paragraph in the README on whether cookies are saved across requests and if it's possible to persist them.

**Comment by B4nan** [comment, marked as answer]
There is no state, this is still just a http client, nothing else, its up to you to manage this. Or use some tool on top of this, e.g. crawlee uses got-scraping internally to do requests with HttpCrawler (or its extensions, e.g. CheerioCrawler), where you have tooling for cookie/session management.

https://crawlee.dev/docs/guides/session-management
